### PR TITLE
data:AFB-M

### PR DESCRIPTION
**Pull request by AFB-M**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5298"></a>

### CODE REVIEW

**Review by imbeshat — COMMENTED** [review]
filename should be ```YourGithubUsername.json```. And please, move your file into ```data``` folder

File: AM [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    },
+     {
+      "name": "Github",
+      "url": "https://github.com/AFB-M",
+      "icon": "FaGithub"
+    },
+     {
+      "name": "Hashnode",
+      "url": "https://afb.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (imbeshat) [inline AM:24]:
```suggestion
      "icon": "SiHashnode"
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by himanshu1221 — APPROVED** [review]
LGTM :+1:

**Review by mahfuztbt — APPROVED** [review]
ready to merge

**Review by Sindhuinti — APPROVED** [review]
Thanks 🎉

**Review by iamthecloverly — APPROVED** [review]

**Pull request merged**